app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtTextActions.java

```diff
@@ -68,7 +68,7 @@ protected ActionCallback getActionCallback(@StringRes int keyId) {
     @Override
     public List<ActionItem> getActiveActionList() {
 
-        final int projectIcon = R.drawable.ic_local_offer_black_24dp;
+        final int projectIcon = R.drawable.ic_add_white_24dp;
```

Comment:
Changed icon to `+` as + is the signifier used for projects.

Reply:
I thought about it as well, but want to keep the `tag` symbol. Plus is too much familar to attachments and alike

Reply:
```suggestion
        final int projectIcon = R.drawable.ic_local_offer_black_24dp;
```